components/forms/w-textfield.vue

```diff
@@ -32,6 +32,10 @@ const inputClasses = computed(() => ({
   [ccInput.suffix]: slots.suffix,
   [ccInput.prefix]: slots.prefix,
 }));
+
+const inputWithPrefixStyle = computed(() => (
+  slots.prefix ? 'padding-left: var(--w-prefix-width, 40px);' : undefined
```

Comment:
Should it be a hard coded fallback here? 🤔 What happens if you don't have that fallback and the `--w-prefix-width` var is not set? Will it then use the `pl-40` from component-classes? 
Having a fallback here would mean that the pl-40 from input prefix in component-classes is redundant, right? 😄

Reply:
> What happens if you don't have that fallback and the --w-prefix-width var is not set? 
Here we are using inline styles, not utility classes, so if prefix exists and the CSS var is not defined, we always will fall back to `padding-left: 40px;`. If however the CSS var was defined, inline styles would still have higher specificity than our `pl-40` component class. So I believe `ccInput.prefix` from line 33 can be removed.

Reply:
Ah ok, I would have expected a rule containing only a missing css variable (without fallback) to be scrapped, but it actually reverts to setting the padding to 0. 🙄😄 👍